Kitodo/src/main/java/org/kitodo/production/services/command/AddDataScript.java

```diff
@@ -36,7 +36,7 @@ public void executeScript(LegacyMetsModsDigitalDocumentHelper metadataFile, Proc
 
         for (String value : metadataScript.getValues()) {
             MetadataEntry metadataEntry = new MetadataEntry();
-            metadataEntry.setKey(metadataScript.getGoal());
+            metadataEntry.setKey(metadataScript.getMetadataKey());
             metadataEntry.setValue(value);
             metadataEntry.setDomain(MdSec.DMD_SEC);
```

Comment:
I would suggest making the domain configurable as a parameter, or better yet, load the ruleset and look it up.